client/src/components/home/image-slider.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import { useRef } from 'react';
+
+import Image from 'next/image';
+
+import { motion, useTransform, useScroll, useInView } from 'framer-motion';
+
+const images = [
+  {
+    src: '/images/hero.jpg',
+    alt: 'Map demo',
+  },
+  {
+    src: '/images/se.jpg',
+    alt: 'Scientific Evidence demo',
+  },
+  {
+    src: '/images/network.jpg',
+    alt: 'Network demo',
+  },
+  {
+    src: '/images/practices.jpg',
+    alt: 'Practices demo',
+  },
+  {
+    src: '/images/datasets.jpg',
+    alt: 'Datasets demo',
+  },
+];
+
+const INITIAL_X = 100;
+const IMAGE_WIDTH = 463;
+
+const ImageSlider = () => {
```

Comment:
You need to delay the animation and/or slow it down because we barely have the time to see the first images.

I think Lidia's intention was to have the images scroll horizontally as you scroll the page (a 1:1 match). I think it's ok if it's just a sliding animation but we need to be able to see all the modules.